### PR TITLE
Show Command and major UI change

### LINKED ISSUE #269: [PE-D] Show command: extra parameters ignored
similar to one of the issues for 'help', after typing 'show', if a user continues entering parameters or any text after it, like 

the show popup window will still appear. Hence the garbage characters are ignored. Perhaps it might be better to include an error message.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: eugenetyc/ped#8

### LINKED ISSUE #270: [PE-D] Show command not working
After a successful delivery, the reports shown on the pop=up window from 'show' should reflect the previous impacts. However, currently it is not updating. Nice design though!

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: eugenetyc/ped#7

### PR DESCRIPTION
**Pull request by Amoscheong97**
Things done:
1) Refactor Show Command
2) Change UI : OrderCard, ReturnCard (Add icons) and add Tabs for statistics and list of orders.
3) Add authors for my classes
4) Edit UG and DG for Show Command
5) Add Test cases
6) Change regex for Cash On Delivery to only allow a value of less than $10,000,000,000,000
7) Edit isSameParcel method to only check for same Transaction ID

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ShowCommand.java [hunk 1]
```diff
@@ -1,21 +1,273 @@
...
+                        ? (startDate.compareTo(dateNow) == 0
+                        ? SHOW_MESSAGE + MESSAGE_TODAY
+                        : SHOW_MESSAGE
+                        + MESSAGE_ONE_DATE
+                        + startDate.format(DATE_FORMATTER))
+                        : SHOW_MESSAGE + MESSAGE_INCLUSIVE
+                        + startDate.format(DATE_FORMATTER)
+                        + TO + endDate.format(DATE_FORMATTER);
+
+                isCommandSuccessful = true;
```
Comment (khsc96) [inline src/main/java/seedu/address/logic/commands/ShowCommand.java]: Probably can SLAP abit harder here, other than that looks good!

File: src/main/java/seedu/address/logic/commands/ShowCommand.java [hunk 2]
```diff
@@ -1,21 +1,273 @@
...
+                if (arrOfDate[0].equals("") || arrOfDate.length > 2) {
+                    throw new ParseException(ILLEGAL_ARGUMENT);
+                }
+
+                if (arrOfDate.length == 2) {
+                    // `show today today` command is not allowed
+                    if (arrOfDate[0].equals(arrOfDate[1]) && arrOfDate[0].equals(TODAY)) {
+                        throw new ParseException(ONE_TODAY_IS_ENOUGH);
+                    }
+                }
```
Comment (khsc96) [inline src/main/java/seedu/address/logic/commands/ShowCommand.java]: Maybe here can drag out as a method? something like
`checkValidInput(arrOfDate)`

File: src/main/java/seedu/address/logic/parser/InsertCommandParser.java [hunk 3]
```diff
@@ -33,6 +33,8 @@
 
 /**
  * Parses input arguments and creates a new InsertCommand object
+ * @author Amoscheong97
```
Comment (khsc96) [inline src/main/java/seedu/address/logic/parser/InsertCommandParser.java]: I think the @author right you got to put above the javadoc or below, you can have a look at either mine or jeremy's classs you need to use the 
`//@@author Amoscheong97` something like this

**Review by khsc96 — CHANGES_REQUESTED** [review]
Overall goodjob! minor changes like the way you type tag author need to change i don't think u can mix it in javadoc cause reposense won't sense until.

**Review by khsc96 — APPROVED** [review]
NAISE USE OF SLAP, CODE IS OBEDIENT NOW! Help to do the merge conflict than it's LGTM! thanks

**Review by JeremyLoh — APPROVED** [review]
LGTM! Thanks!

**Review by Exeexe93 — APPROVED** [review]
LGTM

**Pull request merged**